### ISSUE TITLE
[FEAT] `arm/v8` support? (Raspberry Pi 4)

### ISSUE THREAD

**Issue by FdelMazo** [linuxserver/docker-sonarr#249]
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

Trying to install the image on my Raspi 4 gives me this error, which didn't happen on `linuxserver/radarr` (I'm guessing because radarr has the `Dockerfile.armhf` file, though I'm not sure).

```
Feb 12 23:45:30 sh[2251]: latest: Pulling from linuxserver/sonarr
Feb 12 23:45:30 sh[2251]: no matching manifest for linux/arm/v8 in the manifest list entries
```

I was wondering if there's a specific reason on why this arch is not supported, or if it was just a lack of priority/time.

### Reason for change

Having this working side-by-side with `linuxserver/radarr` would be ideal!

**Comment by Roxedus** [comment]
We support raspberry pi 4 just fine, provided it's not running on a arm32 bit os.

**Comment by FdelMazo** [comment]
Oh, I'm not sure on why my raspi was pulling the wrong image. I ended up hardcoding the `--platform linux/arm64` to the `docker pull` and `docker run` and it ended up working!

(For future reference, I'm on LibreElec 11 using [this](https://github.com/FdelMazo/libreelec-addons/blob/master/docker.linuxserver.sonarr/bin/docker.linuxserver.sonarr) addon which pulls this docker image, where I added the `--platform` arg)

Sorry for opening this (pointless) issue!
